### DISCUSSION TITLE
How can I configure to use it with caddy?

### DISCUSSION THREAD

**Discussion by GerardPolloRebozado** [llamafilm/tesla-http-proxy-addon#140] [General]
dont know where the public key is stored theres a certificates directory but none of its contents is the same public key shown on chrome
i currentyl have this config:
tsl.pollorebozado.com {
        reverse_proxy 192.168.0.221:4430
}

**Comment by 3sv** [comment]
I did a setup with Caddy and the proxy in docker, have a look here:  https://github.com/iainbullock/tesla-http-proxy-docker/issues/58

**Reply by GerardPolloRebozado** [reply]
why you are using the http proxy as a separate docker instead of the HA addon? the addon is not compatible with caddy?
